plugins/CoreHome/vue/src/PeriodSelector/PeriodSelector.vue

```diff
@@ -464,6 +476,44 @@ export default defineComponent({
 
       this.setPiwikPeriodAndDate(this.selectedPeriod, this.dateValue!);
     },
+    updateComparisonValuesFromHash() {
```

Comment:
The method name is a little confusing for me, it mentions 'FromHash' but I don't see hash anywhere within the function or passed as a parameter.
Could it be made clearer?

Reply:
I migrated the naming/watch away from the hash to the (now) single function used from the `ComparisonStore`.